src/py/flwr/dataset/utils/common_test.py

```diff
@@ -53,8 +54,8 @@ def assert_identity(xy_0: XY, xy_1: XY) -> None:
 
 
 class ImageClassificationPartitionedTestCase(unittest.TestCase):
-    """Tests for Partitioned Dataset Ggeneration in Image Classification such
-    as CIFAR-10/100."""
+    """Tests for Partitioned Dataset Generation in Image Classification such as
```

Comment:
Ideally, finish the first sentence on the first line within 80 characters and add further details in subsequent sentences